syft/generic/object_storage.py

```diff
@@ -71,15 +71,22 @@ def get_obj(self, obj_id: Union[str, int]) -> object:
         Returns:
             Object with id equals to `obj_id`.
         """
-
+        obj = None
         try:
-            obj = self._objects[obj_id]
+            obj = self._objects.get(obj_id, None)
+            if obj is None:
```

Comment:
I don't think we should handle placeholders in a different fashion than tensors. Do you know why we need something special for placeholders?

Reply:
Oh, is it because of `PlaceholderId`?

Reply:
yep... but I fixed up the code a bit to make it hopefully cleaner.